src/Test.UnitTests.Sarif.Driver/DriverExtensionMethodsTests.cs

```diff
@@ -53,8 +54,8 @@ private class ValidateSingleFileOutputOptionsTestCase
             public bool ExpectedForce;
         }
 
-        private static readonly ValidateSingleFileOutputOptionsTestCase[] s_validateSingleFileOutputOptionsTestCases =
-            new[]
+        private static readonly ReadOnlyCollection<ValidateSingleFileOutputOptionsTestCase> s_validateSingleFileOutputOptionsTestCases =
```

Comment:
>ReadOnlyCollection [](start = 32, length = 18)

I'm being as immutable as possible these days. Now the list elements can't be replaced. Of course the list elements themselves (the test case objects aren't immutable and IMO it's not worth the trouble to make them. In [C# 9](https://devblogs.microsoft.com/dotnet/welcome-to-c-9-0/?utm_source=vs_developer_news&utm_medium=referral) we'll have immutable records.